src/lib.rs

```diff
@@ -360,6 +391,31 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_points() {
+        // point(x, y)
+        let wkt = <Wkt<f64>>::from_str("POINT (10 20.1)").ok().unwrap();
+        assert!(matches!(wkt.item, Geometry::Point(_)));
```

Comment:
Why not

```rust
        assert!(matches!(
            wkt.item,
            Geometry::Point(Point(Some(Coord {
                x: 10.0,
                y: 20.1,
                z: None,
                m: None
            })))
        ));
```

Reply:
warning: floating-point types cannot be used in patterns
   --> src/lib.rs:402:20
    |
402 |                 y: 20.1,
    |                    ^^^^
    |
    = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
    = note: for more information, see issue #41620 <https://github.com/rust-lang/rust/issues/41620>


Ugh, I see, okay.

Reply:
Wait, the warning was just removed in https://github.com/rust-lang/rust/pull/116284.

Can you leave the test as.. whatever (even the initial `format!` version) and add a comment to change it to a `matches!` when we bump the MSRV to 1.78? Sorry for the back and forth here!

Reply:
So, do we need to change back to string checking?

Reply:
Yeah, let's do that if it's not too much of a hassle :facepalm:.